lib/utils.py

```diff
@@ -108,6 +107,21 @@ def set_extras_n_fix_units(
 
     args["resource_provides_quoted"] = [fixquote(x) for x in args["resource_provides"]]
 
+    # if the user defined the usage_model on the command line,
+    # we need to use their definition of usage_model, not ours
+    # also, we need to define the sites as 'Fermigrid' if they want to run locally
+    for r in args["resource_provides_quoted"]:
+        print(f"r is: {r}")
```

Comment:
I'm not sure what "r" is supposed to indicate to the user.  Perhaps say "resource_provides is {r}" or something like that (if I'm understanding correctly)?

Reply:
That print statement shouldn't be there. That was for my testing. I will remove it.

Reply:
done